qa/vagrant/src/test/resources/packaging/scripts/40_rpm_package.bats

```diff
@@ -49,6 +49,11 @@ setup() {
     [ "$count" -eq 1 ]
 }
 
+@test "[RPM] rpm is signed by our dummy key" {
+    [ "$(cat hostOS)" == "darwin" ] && skip "OSX can't sign the rpm"
+    rpm --checksig elasticsearch*.rpm | grep 'rsa' | grep 'OK'
```

Comment:
AHK - no. Wrong. This will pass for bad sigs because they are logged as `NOT OK`.... Ooops. I'll fix it.